### PR TITLE
CVideoLibraryRefreshingJob: ditch m_showDialogs in favor of CProgressJob::DoModal/IsModal()

### PR DESCRIPTION
**Pull request by Montellese**
This fixes the progress dialog not showing up when refreshing a video item in the GUI which has been broken since #7306.
Thanks to @tamland for reporting the issue.

### CODE REVIEW

**Comment by Montellese** [comment]
@tamland can you verify that this is working again with this PR?

**Comment by tamland** [comment]
Needs rebase as  #7913 added another use of m_showDialogs

**Comment by mkortstiege** [comment]
> Needs rebase as #7913 added another use of m_showDialogs

Thought so as well but this one is touching another part.

**Comment by Montellese** [comment]
@mkortstiege: Well it gets rid of `m_showDialogs` in favor of the inherited `IsModal()` so it still needs a rebase.

**Comment by mkortstiege** [comment]
Oh, right. Brainfart ;)

**Comment by Montellese** [comment]
Done. jenkins build this please

File: xbmc/video/jobs/VideoLibraryRefreshingJob.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@
 #include "guilib/GUIKeyboardFactory.h"
 #include "guilib/GUIWindowManager.h"
 #include "media/MediaType.h"
-#include "utils/log.h"
+#include "utils/log.h"m
```
Comment (tamland) [inline xbmc/video/jobs/VideoLibraryRefreshingJob.cpp]: typo;)

**Comment by Montellese** [comment]
jenkins build this please

**Comment by tamland** [comment]
Progress works again, but one quirk: if you cancel the keyboard when entering name or the movie select dialog, it'll flash a blank progress dialog (just saying "100%") after it was canceled. Probably it isn't marked finished early enough somewhere?

**Comment by Montellese** [comment]
@tamland: Thanks for the report. It was not that I forgot to mark it as finished but that I forced it to 100% before marking it as finished which opened the progress dialog again if it was already closed. Please give it another try.

**Comment by tamland** [comment]
Looks good. thanks!

**Comment by Montellese** [comment]
jenkins build and merge

**Pull request merged**